### DISCUSSION TITLE
Through hole capacitor size

### DISCUSSION THREAD

**Discussion by dwbridgfoot** [bwack/C64-Saver-bwack#39] [General]
Hi, is it possible to use a lower voltage rated capacitor to get a lower seated height? 
The Kessler referenced cap in the BOM is 50v and 11mm seated height. If using a Hammond 1551g case with a 1.6mm pcb, it looks like you will only have 10.4mm clearance. 
I can only find caps on DigiKey less than 11mm if they have a lower rated voltage (16v, 25, 35v etc.).

**Comment by jdecupere** [comment]
Yes that should be OK. I made a version with 25V cap instead f(7mm height) for similar reason - amongst also reducing the size of the board by going to a minimal configuration w/o the provisions for current monitoring etc..
It is not that I have tested it to the point where the “raw” supply would go very high, but in any case that is a (very) abnormal situation and whether or not that cap fails in such case then is not your main issue ;-)

**Reply by dwbridgfoot** [reply]
Thanks, I did see in a video someone use a 25V cap but just wanted to confirm.
